saleor/graphql/product/dataloaders/products.py

```diff
@@ -511,3 +511,15 @@ def batch_load(self, keys):
             collections_channel_listings_by_collection_and_channel_map.get(key, None)
             for key in keys
         ]
+
+
+class CategoryChildrenByCategoryIdLoader(DataLoader):
+    context_key = "categorychildren_by_category"
+
+    def batch_load(self, keys):
+        categories = Category.objects.filter(parent__isnull=False)
```

Comment:
Why not ` Category.tree.root_nodes()` ? It looks like you have a method to fetch parent categories.